core/common/src/main/java/alluxio/cli/AbstractShell.java

```diff
@@ -43,6 +44,7 @@
    *
    * @param commandAlias replacements for commands
    */
+  @SuppressWarnings("nullness")
```

Comment:
@aaudiber @yupeng9 If you remove the warning suppression you will get the following error: 

```[method.invocation.invalid] call to loadCommands() not allowed on the given receiver.
[ERROR] found   : @UnderInitialization(java.lang.Object.class) @NonNull AbstractShell
[ERROR] required: @Initialized @NonNull AbstractShell
```
From the perspective of the Checker Framework, the object may be ```@UnderInitialization``` because it's been initialized with the method ```loadCommands()``` which it's part of the same class. I try to disable this kind of errors using the build flag -AsuppressWarnings=uninitialized. However, at the moment the Checker Framework has an open issue with this kind of errors:
https://github.com/typetools/checker-framework/issues/1590
https://stackoverflow.com/questions/46773276/how-to-suppress-all-initialization-errors

Reply:
Could we fix it by passing the commands in through the constructor instead of using an overridden method? Or add a TODO for this